main/src/test/java/org/mobilitydata/gtfsvalidator/validator/NoticeDocumentationTest.java

```diff
@@ -0,0 +1,89 @@
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mobilitydata.gtfsvalidator.notice.Notice;
+
+@RunWith(JUnit4.class)
+public class NoticeDocumentationTest {
+  private static Pattern MARKDOWN_NOTICE_SIMPLE_CLASS_NAME_ANCHOR_PATTERN =
+      Pattern.compile("^<a name=\"(\\w+(Error|Notice))\"/>");
+
+  private static Pattern MARKDOWN_NOTICE_CODE_HEADER_PATTERN =
+      Pattern.compile("^### (([a-z0-9]+_)*[a-z0-9]+)");
```

Comment:
Do we allow numbers in notice names? (or do want to allow?)

Reply:
We do because we have notices with numbers in their name; `feed_expiration_date7_days` and `feed_expiration_date30_days` to be specific.